app/src/entities/compactMessage/api/searchCompactMessages.ts

```diff
@@ -0,0 +1,18 @@
+import { compactMessagesMock } from '@entities/compactMessage/mock/compactMessagesMock';
+import { CompactMessage } from '@entities/compactMessage/model/compactMessage';
+
+import { getPaginateArray } from '@shared/lib/utils/getPaginateArray';
+
+export const searchCompactMessages = async (
```

Comment:
```suggestion
export const searchCompactMessages = (
```
лучше внутри оберни пока в Promise.resolve